package.json

```diff
@@ -49,17 +49,19 @@
     "coveralls": "^2.11.12",
     "eslint": "^3.3.1",
     "eslint-config-airbnb": "^10.0.1",
-    "eslint-config-airbnb-base" : "^5.0.2",
+    "eslint-config-airbnb-base": "^5.0.2",
     "eslint-plugin-import": "^1.13.0",
     "eslint-plugin-jsx-a11y": "^2.1.0",
     "eslint-plugin-react": "^6.1.1",
     "istanbul": "^0.4.4",
+    "jszip": "^3.1.2",
```

Comment:
Hmm. Is removed in the `dependencies` section below but appears here in the `devDependencies`. Might be by accident?!